isaacConfig.cmake

```diff
@@ -0,0 +1,141 @@
+# - Config file for the isaac package
+# It defines the following variables
+#  isaac_INCLUDE_DIRS - include directories for FooBar
+#  isaac_LIBRARIES    - libraries to link against
+#  isaac_DEFINITIONS  - necessary definitions
+#
+# It defines the following options
+#  ISAAC_THREADING
+#  ISAAC_SHOWBORDER
+#  ISAAC_CUDA
+#  ISAAC_ALPAKA
+#  ISAAC_JPEG
+
+###############################################################################
+# isaac
+###############################################################################
+cmake_minimum_required (VERSION 2.6)
+cmake_policy(SET CMP0048 OLD)
+project (isaac)
+
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu")
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu/isaac")
+
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-std=c++11")
```

Comment:
There is no nice way I know to use `CXX_STANDARD` for header only libs cmake config. It would be possible to define a list of necessary features I call `target_compile_features(...)` with this list.